scripts/release/generate_history_notes.py

```diff
@@ -61,8 +61,9 @@ def get_component_dict():
 def generate_history_notes():
     get_component_dict()
     dev_commits = get_commits()
-    print("Get PRs for {} commits.".format(len(dev_commits)))
+    print("Get PRs for {} commits.".format(len(dev_commits)))
     for commit in dev_commits:
+        print("{} {}".format(commit['sha'], commit['subject']))
```

Comment:
When error occurs, `generate_history_notes.py` fails with no indication of getting information for which commit fails:

```
2021-09-03T02:07:18.6031217Z Traceback (most recent call last):
2021-09-03T02:07:18.6031988Z   File "/home/vsts/work/r1/a/azure-cli/scripts/release/generate_history_notes.py", line 255, in <module>
2021-09-03T02:07:18.6032367Z     generate_history_notes()
2021-09-03T02:07:18.6033036Z   File "/home/vsts/work/r1/a/azure-cli/scripts/release/generate_history_notes.py", line 69, in generate_history_notes
2021-09-03T02:07:18.6033427Z     process_pr(prs[0])
2021-09-03T02:07:18.6033982Z   File "/home/vsts/work/r1/a/azure-cli/scripts/release/generate_history_notes.py", line 174, in process_pr
2021-09-03T02:07:18.6034553Z     search_result = re.search(r'\*\*History Notes\*\*(.*)---',
2021-09-03T02:07:18.6035143Z   File "/opt/hostedtoolcache/Python/3.9.6/x64/lib/python3.9/re.py", line 201, in search
2021-09-03T02:07:18.6391355Z     return _compile(pattern, flags).search(string)
2021-09-03T02:07:18.6392758Z TypeError: expected string or bytes-like object
2021-09-03T02:07:18.6639679Z ##[error]Script failed with error: Error: The process '/bin/bash' failed with exit code 1
```

This PR prints commit ID and subject while getting information for each commit.